backend/app/processing/llm.py

```
@@ -6,10 +6,12 @@
 from core.logger import logger
 
 
+CONTENT_TYPE_JSON = "application/json"
+
 def create_prompt(query: str, context_docs: list, conversation_history=None) -> str:
```

Ruff (0.8.2)

Missing type annotation for function argument `conversation_history`

(ANN001)